versus-fe/src/containers/LoginPage/LoginPage.js

```diff
@@ -1 +1,65 @@
 
+import React, { Component } from 'react';
+import './LoginPage.css'
+
+class LoginPage extends Component {
+
+  constructor(props){
+    super(props)
+    this.state = {
+      username: '',
+      password: ''
+    }
+  }
+
+  sendDetails = (event) => {
+    event.preventDefault()
+
+  let loginDetails = {
+      username:this.state.username,
+      password:this.state.password
+    }
+
+    fetch('http://localhost:3000/loginDetails', {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json; charset=utf-8"
+      },
+      body: JSON.stringify(loginDetails)
+    })
+
+  }
+
+  handleUsernameChange =  (event) => {
```

Comment:
Use a single function `handleFormChange` for both. Use the `name` html attr for the inputs.